### PR TITLE
fixes to header, footer, and submit button

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
R?

File: components/footer.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ var Footer = React.createClass({
         <div className="footer">
           <div className="row">
             <div className="half">
-              <img src="/images/Mozilla-wordmark.png"/>
+              <img src="/images/mozilla.svg"/>
```
Comment (ScottDowne) [inline components/footer.jsx]: Updated to new image file provided by Jordan.

It's and svg, smaller, and is a bit lighter in colour.

File: less/new-flow-test/footer.less [hunk 2]
```diff
@@ -0,0 +1,12 @@
+footer {
+  .footer {
```
Comment (ScottDowne) [inline less/new-flow-test/footer.less]: Make the footer in the middle just for the new flow pages.

File: less/new-flow-test/shared.less [hunk 3]
```diff
@@ -102,8 +102,6 @@ input[name="payment-type"] + label {
 input[name="payment-type"] + label {
   background-image: linear-gradient(transparent, rgba(0, 0, 0, 0.2));
   position: relative;
-  padding-top: 5px;
-  padding-bottom: 5px;
```
Comment (ScottDowne) [inline less/new-flow-test/shared.less]: The padding on number buttons looks centered, but with lower case text, it looked off.

So I'm splitting the padding bit out of the rule, and into separate specific ones.

File: public/images/mozilla.svg [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (alicoding) [inline public/images/mozilla.svg]: There seems to be a problem with this image... check this out https://github.com/ScottDowne/donate.mozilla.org/blob/footer-pos/public/images/mozilla.svg

**Comment by alicoding** [comment]
Fix the image then r+

File: public/images/mozilla.svg (hunk 4 above)
Comment (ScottDowne) [inline public/images/mozilla.svg]: It's just something weird with github's display, right click the image in the above link and click view image, should be fine.

**Comment by alicoding** [comment]
you need to rebase then merge this

**Comment by ScottDowne** [comment]
rebased

**Pull request merged**